### DISCUSSION TITLE
More troubles with integrate forces and direct manipulation.

### DISCUSSION THREAD

**Discussion by yosoyfreeman** [godot-jolt/godot-jolt#639] [Questions]
Hi there! Not so long ago i opened this issue which involved unexpected behaviors while manipulating bodies: https://github.com/godotengine/godot/issues/79835 it was later fixed. However, i noticed that still, trying to use  this features keep causing the body to break. This post was opened a few days ago, which involve a similar issue just by reading, not modifying the values of the body: https://github.com/godotengine/godot/issues/83029

I was going to try and isolate this issues as bugs and report them, but this issues seem related to each other, so i wonder if it may be something in common causing this kind of issues, that's why opened this conversation.  I know is not Jolt specific, but since Godot has no physics maintainers right now and you are the main figure pushing for it i thought it would be worth to ask you directly.  I fear there is something broken on the Godot side, and as such affecting Godot Jolt too and its future adoption. 

If you think there is nothing wrong and this are isolated issues on the Godot side i will try to isolate more of them and report them to Godot. I just wanted to know your opinion to know the best way to proceed. Thank you for your time!

**Comment by mihe** [comment]
See my comment in godotengine/godot#83029.
